tests/index.e2e.ts

```diff
@@ -0,0 +1,147 @@
+import StoryblokClient from '../src/'
```

Comment:
@alvarosabu while in the unit tests is perfectly fine, as this is an integration test (api tests to be more specific) it shouldn't use the src, but rather the compiled version of the client

Reply:
Good point. Thanks